junit-platform-gradle-plugin/src/test/groovy/org/junit/platform/gradle/plugin/JUnitPlatformPluginFunctionalSpec.groovy

```diff
@@ -208,7 +209,7 @@ class AdderTest {
 		}
 }
 '''
-			)
+					)
```

Comment:
:+1: 
GrEclipse is not that stable when it comes to multiline strings. After try&error I found that it handled this occurrence differntly since [its beginning](https://github.com/fvgh/junit5/blob/46a1937feccbc456b7e05f8db9803037ac3e14f8/junit-platform-gradle-plugin/src/test/groovy/org/junit/platform/gradle/plugin/JUnitPlatformPluginFunctionalSpec.groovy#L197) did not start with a new line. After I changed that, it formatted the parentheses  correctly.